bokeh/command/subcommands/serve.py

```diff
@@ -198,7 +198,7 @@ def invoke(self, args):
 
         server_kwargs = { key: getattr(args, key) for key in ['port',
                                                               'address',
-                                                              'keep_alive']
+                                                              'keep_alive_milliseconds']
                           if getattr(args, key, None) is not None }
```

Comment:
me neither. remember the origin of it was to use kwarg default values instead of "None means default". So we can't pass down kwarg that we want to have defaulted.